src/autoFetcher.js

```diff
@@ -132,6 +132,34 @@ AutoFetcher.prototype.justFetch = function(urls) {
   this.worker.postMessage({ type: 'fetch-all', values: urls });
 };
 
+
+/**
+ * Sends the supplied url with extra options to indicate
+ * that this is a page to backing worker
+ * @param {Array<string>} url
+ */
+AutoFetcher.prototype.fetchAsPage = function(url, originalUrl, title) {
+  if (!url) {
+    return;
+  }
+
+  var headers = {"X-Wombat-History-Page": originalUrl}
+  if (title) {
+    title = encodeURIComponent(title.trim());
```

Comment:
It would be clearer to create a named variable representing the encoded title.
Also not a good idea to re-assign function arguments ([reference](https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/)) 
```suggestion
    var encodedTitle = encodeURIComponent(title.trim());
```